### PR TITLE
Version 0.12.0

### LINKED ISSUE #286: [Bug Report] Mod does not show up when searching for its database name.
### What Happened?

Mod does not show up when searching for its database name.
Example of this would be [No Alt Tab Pause](https://outerwildsmods.com/mods/noalttabpause/).
It only appears when you search its manifest name, which is Run in Background.
This is confusing as the name shown is the database name (legacy manager would show manifest name).
It should appear for both the database and manifest names even though both should be the same, but this is a special case.
Searching for mod
No results

### Package

Graphical Interface

### Platform

Windows

### PR DESCRIPTION
**Pull request by Bwc9876**
## Core Changes

- **BREAKING**: `ProtocolInstallType` has been renamed to `ProtocolVerb`, `ProtocolPayload::install_type` has been renamed to `ProtocolPayload::verb`
- **BREAKING**: `ProgressPayload::parse` now returns a `Result<ProgressPayload>` instead of a `ProgressPayload`, you now need to error handle for when an invalid progress payload is parsed
- **BREAKING**: `ProgressBars::process` now takes an **owned** `ProgressPayload` instead of a `&str`, you now need to handle parsing the progress payload (use `ProgressPayload::parse`)
- **BREAKING**: `UnsafeLocalMod::Valid` is now placed in a `Box` for less memory usage
- Added new `run-game` protocol verb
- Added New `thumbnail` field to `RemoteMod` and subsequent `ModThumbnail` struct
- Added New `search_with_remote` method for `LocalDatabase`, which searches the local database while taking remote db names into account
- Add `donate_link` property to `ModManifest` for donation links (https://github.com/ow-mods/owml/pull/561)
- Derive `Debug` on most structs for easier debugging

## CLI Changes

- Added `raw` command for displaying database contents as JSON
- Added `--assert-setup` to exit with code 2 if OWML isn't setup
- Added support for `run-game` protocol verb
- Added `donation_link` display in `owmods info`

## GUI Changes

- Added thumbnails to mod rows (can be disabled in settings)
- Added a settings toggle to hide mods in the remote tab that require the DLC
- Added a puzzle piece icon next to mod names that require the DLC
- Added the ability to launch the game using the `owmods` URI
- Added a "Donate" button in the overflow menu for mods that specify `donateLink` in their manifest (https://github.com/ow-mods/owml/pull/561)
- Added a button to copy the unique name of a mod to the overflow menu
- Added a button to open the manager's log to the overflow menu
- Better distinction between whether the remote database is still fetching or if there was an error while fetching it.
- Better distinction when loading a remote mod between whether it was in the database, it wasn't in the database, the database is still loading, and the database failed to load.
- Window now requests focus earlier when being invoked by the protocol
- Switch settings to a 2-column layout
- Factor remote database names into local search (resolves #286)
- The log window now has a lower threshold for when logs are incoming too fast
- Remove Ctrl+P for print dialogue

## Other Changes

- The nix flake that was previously in a [separate repo](https://github.com/ow-mods/ow-mod-man-flake) has now been added to this one. To use it see `README.md` in the `nix` folder. 
- The flake now builds the manager from scratch instead of using a `.deb` file. This process takes a while so we're currently working on getting a cachix instance ready for it.

### CODE REVIEW

File: nix/owmods-gui.nix [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 stdenv.mkDerivation rec {
   pname = "owmods-gui";
-  version = "0.11.3";
```
[the remark this answers is not in the document]
Comment (Bwc9876) [reply nix/owmods-gui.nix]: I think we'll just have it be broken for the little bit (like 30 minutes) it takes for the release to build. It's not ideal but I wanna get this version out soon.

**Comment by Bwc9876** [comment]
@xiaojiafei520 Sorry for the pings, this should be the last one for a while. But could you update the `COPY_UNIQUE_NAME`, `HIDE_DLC_MODS`, `TOOLTIP_HIDE_DLC_MODS`, `HIDE_MOD_THUMBNAILS`, `TOOLTIP_HIDE_THUMBNAILS`, and `REQUIRES_DLC` keys?

**Comment by xiaojiafei520** [comment]
@Bwc9876 No problem!

**Comment by xiaojiafei520** [comment]
@Bwc9876 Sorry, I couldn't find these keys in template.json and english.json. Do I need to add them myself?

**Comment by Bwc9876** [comment]
> @Bwc9876 Sorry, I couldn't find these keys in template.json and english.json. Do I need to add them myself?

Strange, I'm able to access them here: https://github.com/ow-mods/ow-mod-man/blob/dev/owmods_gui/frontend/src/assets/translations/english.json

```
"COPY_UNIQUE_NAME": "Copy Unique Name",
"HIDE_DLC_MODS": "Hide DLC Mods",
"HIDE_MOD_THUMBNAILS": "Hide Mod Thumbnails",
"REQUIRES_DLC": "This mod requires the Echoes of the Eye DLC",
"TOOLTIP_HIDE_DLC_MODS": "Hide mods in the remote tab that require the Echoes of the Eye DLC",
"TOOLTIP_HIDE_MOD_THUMBNAILS": "If enabled, mod thumbnails will not be shown.",
```

**Comment by xiaojiafei520** [comment]
@Bwc9876 oh! Sorry,I forgot to update my branch

**Pull request merged**